sumologic-service/0.15.0/README.md

```diff
@@ -0,0 +1,151 @@
+# sumologic-service
+![GitHub release (latest by date)](https://img.shields.io/github/v/release/keptn-sandbox/sumologic-service)
+[![Go Report Card](https://goreportcard.com/badge/github.com/keptn-sandbox/sumologic-service)](https://goreportcard.com/report/github.com/keptn-sandbox/sumologic-service)
+
+This implements the `sumologic-service` that integrates the [Sumo Logic](https://en.wikipedia.org/wiki/Sumo_Logic) observability platform with Keptn. This enables you to use Sumo Logic as the source for the Service Level Indicators ([SLIs](https://keptn.sh/docs/0.15.x/reference/files/sli/)) that are used for Keptn [Quality Gates](https://keptn.sh/docs/concepts/quality_gates/).
+If you want to learn more about Keptn visit us on [keptn.sh](https://keptn.sh)
+
+Check the issue here for more info: https://github.com/keptn/integrations/issues/20
+
+
+**BEFORE YOU START**, please be aware that there are more ways to integrate with your service that don't require creating a service from this template, see https://keptn.sh/docs/0.15.x/integrations/how_integrate/ for more details.
```

Comment:
Remove this boilerplate as it doesn't make sense here

Reply:
Fixed